src/sage/combinat/tableau.py

```diff
@@ -881,6 +987,16 @@ def plot(self, descents=False):
             sphinx_plot(t.plot())
             Tableaux.options.convention="english"
 
+        If Russian notation is set, we tilt the French by 45 degrees:
```

Comment:
```suggestion
        If Russian notation is set, we tilt the French notation by 45 degrees:
```
At least, I don't think you want to make every French person live at a 45 degree angle. `:P`

Reply:
All the French!